prisma/migrations/20220119200600_updated_to_match_schema_changes/migration.sql

```diff
@@ -0,0 +1,21 @@
+/*
```

Comment:
Probably better to have a more descriptive migration name

Reply:
Done in latest commit. Good point lol.